### PR TITLE
@craigspaeth Allow admin to add auction_id to an article

### PR DESCRIPTION
**Pull request by kanaabe**
Autosaves when you click on the auction and retrieves the auction when you come back to page. 

positron_auction_autocomplete

### CODE REVIEW

File: api/apps/articles/model.coffee [hunk 1]
```diff
@@ -65,6 +65,7 @@ schema = (->
   featured_artwork_ids: @array().items(@objectId()).allow(null)
   fair_id: @objectId().allow('', null)
   partner_ids: @array().items(@objectId()).allow(null)
+  auction_id: @objectId().allow('', null)
```
Comment (kanaabe) [inline api/apps/articles/model.coffee]: Better to rename to sale_id?

Comment (kanaabe) [inline api/apps/articles/model.coffee]: Also, a user should only be allowed to add it to a single auction, correct?

File: client/apps/edit/components/admin/test/index.coffee [hunk 2]
```diff
@@ -22,6 +22,7 @@ describe 'EditAdmin', ->
         EditAdmin::setupAuthorAutocomplete = sinon.stub()
         EditAdmin::setupFairAutocomplete = sinon.stub()
         EditAdmin::setupPartnerAutocomplete = sinon.stub()
+        EditAdmin::setupAuctionAutocomplete = sinon.stub()
```
Comment (kanaabe) [inline client/apps/edit/components/admin/test/index.coffee]: I didn't see tests for the other ones, but I'm happy to write them if you see fit!

**Comment by kanaabe** [comment]
Closes https://github.com/artsy/positron/issues/215

File: api/apps/articles/model.coffee (hunk 1 above)
Comment (craigspaeth) [inline api/apps/articles/model.coffee]: Good point... I'm not strongly opinionated on that naming. I like to optimistically think we'll some day refactor auctions to a first class thing, so I'm leaning towards `auction_id`.

For now yeah single auction is fine. We may need to make this a many to many later down the line... but I'll also probably have to address that with fairs—I'll cross that bridge when we get to it.

File: client/apps/edit/components/admin/test/index.coffee (hunk 2 above)
Comment (craigspaeth) [inline client/apps/edit/components/admin/test/index.coffee]: Nah in this case we'd just be testing a bunch of configuration options—not a lot of code to test there.

**Comment by craigspaeth** [comment]
Sweet, thanks! Pretty straight forward :smile_cat:

**Pull request merged**